cmake/modules/AddSwift.cmake

```diff
@@ -719,9 +719,9 @@ function(_add_swift_library_single target name)
           ${SWIFTLIB_SINGLE_SDK} STREQUAL "OSX")
         # HACK: don't build WatchKit API notes for OS X.
       else()
-        if (NOT IS_DIRECTORY "${SWIFT_SOURCE_DIR}/stdlib/public/SDK/${framework_name}")
-          list(APPEND SWIFTLIB_SINGLE_API_NOTES "${framework_name}")
-        endif()
+        # Always build the "non-overlay" apinotes to keep them in sync
```

Comment:
Comment nitpick: this is saying "Always build *all* API notes", not just the non-overlay ones.

Reply:
Well, it's building all the apinotes found in `swift/apinotes/`, but I can adjust the comment.